Kernel_23/doc/Kernel_23/CGAL/intersections.h

```diff
@@ -95,7 +95,7 @@ The following tables give the possible values for `Type1` and `Type2`.
 
 \cgalHeading{2D Intersections}
 
-The return type can be obtained through `CGAL::cpp11::result_of<Kernel::Intersect_2(A, B)>::%type`.
+The return type can be obtained through `cpp11::result_of<Kernel::Intersect_2(A, B)>::%type`.
```

Comment:
... There, do we want to stick with `result_of`, or tell our user to use `auto` and `decltype` instead?

Reply:
I would actually do the latter. In any case, we plan to remove the `result_of` in the near future and it is deprecated starting from C++17 anyway.